### PR TITLE
storage: Disallow duplicate mount points

### PR DESCRIPTION
**Pull request by mvollmer**
UDisks2 doesn't handle them well.

### CODE REVIEW

File: pkg/storaged/fsys-tab.jsx [hunk 1]
```diff
@@ -60,6 +60,29 @@ export function get_fstab_config(block) {
...
+    return blocks;
+}
+
+export function is_valid_mount_point(client, block, val) {
+    if (val === "")
+        return _("Mount point cannot be empty");
+
+    const other_blocks = find_blocks_for_mount_point(client, val, block);
+    if (other_blocks.length > 0)
+        return cockpit.format(_("Mount point already used for $0"),
```
Comment (martinpitt) [inline pkg/storaged/fsys-tab.jsx]: Mount point "is" already used?

File: test/verify/check-storage-mounting [hunk 2]
```diff
@@ -247,5 +247,34 @@ class TestStorage(StorageCase):
...
+        self.content_row_wait_in_col(2, 2, "/dev/test/two")
+
+        # Format the first and give it /run/data as the mount point
+        self.content_head_action(1, "Format")
+        self.dialog({"type": "ext4",
+                     "mount_point": "/run/data"})
+        self.content_row_wait_in_col(1, 1, "ext4 File System")
+        self.content_tab_wait_in_info(1, 2, "Mount Point", "/run/data")
+
+        # Format the secondt and also try to use /run/data as the mount point
```
Comment (martinpitt) [inline test/verify/check-storage-mounting]: second"t" typo `</totally-unimportant>`

**Comment by martinpitt** [comment]
#12647 landed, please rebase.

File: pkg/storaged/fsys-tab.jsx (hunk 1 above)
Comment (mvollmer) [reply pkg/storaged/fsys-tab.jsx]: Yes, fixed.

File: test/verify/check-storage-mounting (hunk 2 above)
Comment (mvollmer) [reply test/verify/check-storage-mounting]: Fixedt.

**Comment by martinpitt** [comment]
Huh, why doesn't this have tests..

**Pull request merged**